src/ark/utils/deepcell_service_utils.py

```diff
@@ -222,34 +253,22 @@ def run_deepcell_direct(input_dir, output_dir, host='https://deepcell.org',
     http.mount('https://', adapter)
     http.mount('http://', adapter)
```

Comment:
@alex-l-kong Do you think I should scrap the retry_strategy stuff and use requests.post() like before? Or would just setting num_retries=1 would be appropriate and keep the upload method the same.

Reply:
I think we should just convert to `requests.post()` for consistency purposes.